src/server.js

```diff
@@ -603,6 +614,8 @@ function getClientConfiguration(network) {
 		config.defaults.nick = Helper.getDefaultNick();
 	}
 
+	config.uploadMaxFileSize = uploader.getMaxFileSize();
```

Comment:
If I read this correctly, when `Helper.config.upload` is `false` (or not set in the config, i.e. coming from defaults which are `false`), `uploader` hasn't been set so it's `undefined` 💥